package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@slice-and-dice/register-a-food-business-validation",
-  "version": "1.18.0",
+  "version": "1.19.0",
   "description": "Validation functions for the register a food business service",
   "main": "index.js",
   "scripts": {
-    "format:verify": "prettier --list-different validationFunctions/**/*.js ./valid-business-types.json",
-    "format": "prettier --write validationFunctions/**/*.js ./valid-business-types.json",
-    "lint": "eslint ./validationFunctions",
```

Comment:
it formats these horribly does it?

Reply:
valid-business-types.json no longer exists, moved into enums folder.

Reply:
sorry yeah - I can't read.